Mathlib/RingTheory/IntegrallyClosed.lean

```diff
@@ -123,16 +123,16 @@ theorem integralClosure_eq_bot_iff : integralClosure R K = ⊥ ↔ IsIntegrallyC
 #align is_integrally_closed.integral_closure_eq_bot_iff IsIntegrallyClosed.integralClosure_eq_bot_iff
 
 @[simp]
-theorem pow_dvd_pow_iff {n : ℕ} (hn : n ≠ 0) {a b : R} : a ^ n ∣  b ^ n ↔ a ∣ b  := by
+theorem pow_dvd_pow_iff {n : ℕ} (hn : 0 < n) {a b : R} : a ^ n ∣  b ^ n ↔ a ∣ b  := by
```

Comment:
I think `n ≠ 0` is preferred over `0 < n` for assumptions.

Reply:
I also prefer it, but I guess it's better to keep all the statements the same. Let me see how painful is to modify the `Int` and `Nat` versions.